### PR TITLE
operator: implement circuitbreaker

### PR DESCRIPTION
**Pull request by fromanirh**
Implement circuitbreaker-like behaviour for the SSP operator.
With is change is now possible to *temporarily* make the operator ignore
one or more operands.
This is intended to be used as extrema ratio parachute
to troubleshoot issue in clusters, when everything else failed and the
state has to be recovered, or at very least investigated, somehow.

see `TROUBLESHOOTING.md` to learn about how to use this
functionality.

Signed-off-by: Francesco Romani <fromani@redhat.com>

### CODE REVIEW

**Comment by fromanirh** [comment]
tested on a live cluster. Works as expected. Not sure about meanongful functests, but they can be added later.

File: roles/KubevirtCircuitBreaker/defaults/main.yml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+---
+# defaults file for KubevirtCircuitBreaker
```
Comment (MarSik) [inline roles/KubevirtCircuitBreaker/defaults/main.yml]: Delete the empty files please.

Comment (fromanirh) [reply roles/KubevirtCircuitBreaker/defaults/main.yml]: Done (here and everywhere else)

**Comment by fromanirh** [comment]
tested on a live cluster even after the last changes. Works as intended.

**Pull request merged**